config/sequins.exs

```diff
@@ -71,10 +73,23 @@ config :sequins, CreatePyramidTiff,
     CreatePyramidTiff: [status: :retry]
   ]
 
+config :sequins, CreateTranscodeJob,
+  queue_config: [
+    receive_interval: 1000,
+    wait_time_seconds: 1,
+    processor_concurrency: 1,
+    visibility_timeout: 300
+  ],
+  notify_on: [
+    CreatePyramidTiff: [status: :ok, role: "A"],
+    CreateTranscodeJob: [status: :retry]
+  ]
+
 config :sequins, FileSetComplete,
   queue_config: [receive_interval: 1000, wait_time_seconds: 1, processor_concurrency: 10],
   notify_on: [
     CreatePyramidTiff: [status: :ok],
     FileSetComplete: [status: :retry],
-    ExtractExifMetadata: [status: :ok, role: "P"]
+    ExtractExifMetadata: [status: :ok, role: "P"],
+    CreateTranscodeJob: [status: :ok]
```

Comment:
Subscribing to both `:ok` and `:skip` from `CreateTranscodeJob` will tie off this PR, and I can change it further it when I add `TranscodeJobComplete`.
```suggestion
    CreateTranscodeJob: [status: :ok],
    CreateTranscodeJob: [status: :skip]
```